Initialtask/migrations/Version20210629225452.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20210629225452 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return '';
+    }
+
+    public function up(Schema $schema): void
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->addSql('ALTER TABLE tblproductdata ADD intStock INT DEFAULT NULL, ADD floatCost DOUBLE PRECISION DEFAULT NULL'
```

Comment:
Please use int type for pricing, and convert to cents during import, to exclude rounding issue